bitnami/nginx/values.yaml

```diff
@@ -245,6 +245,9 @@ containerSecurityContext:
 ##
 containerPorts:
   http: 8080
+  ## Set to "" to disable the standard http port.
+  ## Note that this will disable the default probes and ingresses.
```

Comment:
Please avoid disabling the default probes

Reply:
Comments removed.